packages/common/src/authorization/projectMemberAbility.test.ts

```diff
@@ -1298,7 +1298,7 @@ describe('Project member permissions', () => {
                         'manage',
                         subject('Space', { projectUuid, isPrivate: false }),
                     ),
-                ).toEqual(false);
+                ).toEqual(true);
```

Comment:
I'm a bit worried I had to change these tests. Perhaps we need to change the logic on the promote service